docs/integrations/github/discovery.md

```diff
@@ -53,6 +53,51 @@ And then add the entity provider to your catalog builder:
   }
 ```
 
+## Installation with Events Support
+
+Please follow the installation instructions at
+
+- https://github.com/backstage/backstage/tree/master/plugins/events-backend/README.md
+- https://github.com/backstage/backstage/tree/master/plugins/events-backend-module-github/README.md
+
+Additionally, you need to decide how you want to receive events from external sources like
+
+- [via HTTP endpoint](https://github.com/backstage/backstage/tree/master/plugins/events-backend/README.md)
+- [via an AWS SQS queue](https://github.com/backstage/backstage/tree/master/plugins/events-backend-module-aws-sqs/README.md)
+
+Set up your provider
+
+```diff
+// packages/backend/src/plugins/catalogEventBasedProviders.ts
++import { CatalogClient } from '@backstage/catalog-client';
```

Comment:
Is this line supposed to be added? I don't see a use of the client 🤔

Reply:
Nice catch, forgot to remove after the refactor